process/block/preprocess/intermediateResults.go

```diff
@@ -237,6 +237,25 @@ func (irp *intermediateResultsProcessor) CreateMarshalizedData(txHashes [][]byte
 	return mrsTxs, nil
 }
 
+// GetAllCurrentUsedTxs returns the cached finalized transactions for current round
+func (irp *intermediateResultsProcessor) GetAllCurrentUsedTxs() map[string]data.TransactionHandler {
```

Comment:
GetAllIntraShardGeneratedScResults? as this reflect better what it does

Reply:
this implements an interface. it will not be necessary sc result.